week-1/Homework/extra/1-currency-conversion.js

```diff
@@ -5,8 +5,11 @@
   Write a function that converts a price to USD (exchange rate is 1.4 $ to £)
 */
 
-function convertToUSD() {}
+function convertToUSD(priceInPound) {
+return (priceInPound * 1.4);
+}
```

Comment:
Amazing Laetitia, the function is correct the only thing that you don't need here is the Parentheses.

Reply:
Fixed :)